test/tests/table.ts

```diff
@@ -378,12 +361,15 @@ suite('Table', () => {
                 const allRequestedRows = await cursor.all()
                 assert.equal(allRequestedRows.length, 10000)
             })
-
             test('should stop if requesting more than exists', async () => {
                 const tableCursor = decentiumTrendingTable.first(10000)
                 const firstBatch = await tableCursor.all()
                 assert.equal(firstBatch.length, 239)
             })
+            test('should not fetch more rows than what is requested', async () => {
```

Comment:
Was this supposed to do something? Feels like a fragment that got left behind 😉

Reply:
haha yep that was supposed to be removed 😅 Will clean that up now!